app/views/fact_values/_fact.html.erb

```diff
@@ -24,7 +24,7 @@
   <td class='fact-origin'><%= fact_origin_icon(fact.origin) %></td>
   <td><%= fact_from fact %></td>
   <td><% unless fact.compose %>
-          <%= link_to_function(content_tag(:button, icon_text("adjust", "")),
+          <%= link_to_function(content_tag(:span, "Distribution Chart", :class => "btn btn-xs btn-default", :style => "font-size: 10px;"),
```

Comment:
I'm trying to avoid inline styling, if changes are required later, it's easier for me to look into one css folder. Also if we wanted to reuse CSS variables (SCSS) for font-size, it would not be possible in inline styling. So yes, in such case I'd prefer new CSS class. But I'm probably not the best person to ask :-)